configs/prow-dev/config/plugins.yaml

```diff
@@ -12,7 +12,7 @@ config_updater:
 welcome:
   - repos:
       - tidb-community-bots
-    message_template: "Welcome @{{.AuthorLogin}}! <br><br>It looks like this is your first PR to <a href='https://github.com/{{.Org}}/{{.Repo}}'>{{.Org}}/{{.Repo}}</a> 🎉. <br><br>You will be prompted by a bot to use commands during the review process. Do not be afraid to follow the prompts! It is okay to experiment. [Here is the bot commands documentation](https://prow-dev.tidb.io/command-help). <br><br>You can also check if {{.Org}}/{{.Repo}} has [its own contribution guidelines](https://github.com/{{.Org}}/{{.Repo}}/tree/master/CONTRIBUTING.md). <br><br>We want to make sure your contribution gets all the attention it needs! <br><br>Thank you, and welcome to tidb community. :smiley:"
+    message_template: "Welcome @{{.AuthorLogin}}! <br><br>It looks like this is your first PR to <a href='https://github.com/{{.Org}}/{{.Repo}}'>{{.Org}}/{{.Repo}}</a> 🎉. <br><br>I'm the bot to assist you in requsting reviewrs, tagging labels and more. See [available commands](https://prow-dev.tidb.io/command-help). <br><br>Welcome to ti-community. :smiley:"
```

Comment:
I still think the original welcome message was more friendly and polite.

And `reviewrs`  is a typo.

I don't think "ti-community" is appropriate because the name of the organization is "tidb-community-bots".

Reply:
> I don't think "ti-community" is appropriate because the name of the organization is "tidb-community-bots".

I'd like to replace it by the repo name.

> I still think the original welcome message was more friendly and polite.

The original is puzzling to me...

Reply:
> I'd like to replace it by the repo name.

This message will only be replied to in the first pr of this organization, so I think using the repo name may not be appropriate.